Sources/Public/Animation/DotLottie/DotLottieFile.swift

```diff
@@ -0,0 +1,119 @@
+//
+// DotLottieFile.swift
+// LottieFiles
+//
+// Created by Evandro Harrison Hoffmann on 27/06/2020.
+// Copyright © 2020 LottieFiles. All rights reserved.
+//
+
+import Foundation
+
+/// Detailed .lottie file structure
+public struct DotLottieFile {
+  public let localUrl: URL
+  
+  public static let manifestFileName: String = "manifest.json"
```

Comment:
Ideally we'd limit the set of new public APs to only include types / methods / properties that are strictly necessary for consumers of the library. For example, in this file it seems like everything should be internal other than the initializers.